.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,16 @@
+{
+  "ExpandedNodes": [
```

Comment:
Also, add these files in gitignore if it is not important. so we don't have to push these changes to current codebase

Reply:
sure, I will add it to gitignore

Reply:
@rijusougata13 I tried to add this file in gitignore but it's not getting ignored. So, I found one solution from the internet to clear the catch, but It will cause clearing cache for all files available.